src/collections/blog/2025/03-14-Getting-started-with-Meshrey/aadityaa.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: "Getting Started with Meshery: A Guide for Platform Engineers"
+date: 2025-03-14
+author: Aditya Raj
+thumbnail: ./aadi.png
+darkthumbnail: ./aadi.png
+category: "Service Mesh"
```

Comment:
We are not using 'Service Mesh' if I am correct.

Reply:
What's with the image filename?

Reply:
Why would this be under Service Mesh?